packages/react-native-win32/package.json

```diff
@@ -13,9 +13,9 @@
     "lint:fix": "just-scripts lint:fix",
     "lint": "just-scripts lint",
     "api": "just-scripts api",
-    "run-win32-dev-web": "rex-win32 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev --useDevMain --useWebDebugger",
-    "run-win32-devmain": "rex-win32 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev --useDevMain",
-    "run-win32": "rex-win32 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev",
+    "run-win32-dev-web": "npx @office-iss/rex-win32@0.62.7-tenantreactnativewin-13222 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev --useDevMain --useWebDebugger",
+    "run-win32-devmain": "npx @office-iss/rex-win32@0.62.7-tenantreactnativewin-13222 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev --useDevMain",
+    "run-win32": "npx @office-iss/rex-win32@0.62.7-tenantreactnativewin-13222 --bundle RNTester/js/RNTesterApp --component RNTesterApp --basePath ./dist/win32/dev",
     "start": "react-native start",
```

Comment:
>start [](start = 5, length = 5)

So we have "start" and "run*" can naming be unified?

Reply:
They do different things, and are aligned with react-native's use of start and run.